### ISSUE TITLE
orange pi one  - desire headless login - no WiFi sensed

### ISSUE THREAD

**Issue by thadG** [MichaIng/DietPi#1400]
## Creating a  report/issue:
Followed directions, as per:
http://dietpi.com/phpbb/viewtopic.php?f=8&t=9#p9
Followed Step1, Step2, optional wifi, and Step 3
does not create WiFi connection

#### Required Information:
- DietPi Version |   DietPi_OrangePiOne-armv7-(Jessie).img
- SBC Device (EG: RPi 3)?Orange Pi One
- Power supply used -   5v 4A power
- SD card used (Pendrive  micro SD, HC,  8GB)?
- Distro (EG: Jessie) | Jessie

#### Additional Information (if applicable):
- Software title?
- Can this issue be replicated on a fresh installation of DietPi?  YES
tried several times, results the same
- tring to connect headless (without Ethernet cable, only wireless connection)

Edited dietpi.txt using notebook:::
 
```
Ethernet_Enabled=0
Wifi_Enabled=1

Wifi_SSID=myRouterVALUE  // tried with and without quotes around myRouterVALUE
Wifi_KEY=myRouterPASSWORD   // tried with and without quotes around myRouterPASSWORD

Use_Static=1
Static_IP=192.168.1.211
Static_Mask=255.255.255.0
Static_Gateway=192.168.1.1
```

Write dietpi.txt, then place SD card into Orange Pi One.
Apply power, wait,  try to SSH to Orange Pi One
#### Expected behaviour:

Be able to use KiTTy with host name 192.168.1.211 and port 22
so I can  login wirelessly into Orange Pi One.
My router does "see" 192.168.1.211
 
#### Actual behaviour:

KiTTy opens window but does not respond

#### Steps to reproduce:

The directions above
#### Did you submit a ```dietpi-bugreport```?

Is this not the bug report as well?

#### Extra details:

Basically, I want to login into Orange Pi One headless. And only use WiFi
as means of connection.

**Comment by Fourdee** [comment]
@thadG 

Hi,

Thanks for the report 👍 

Please can you try the v6.0 RC image?:
http://dietpi.com/downloads/testing/v6.0/DietPi_OPiOne-ARMv7-Stretch.7z

**Comment by thadG** [comment]
Hi Dan,

Tried the image, which is not partitioned. Cannot get to the dietpi.txt
file to edit.
But tried it anyway, and cannot operate it  headless as no WiFi node
presents an IP address .

Thad

**Comment by Fourdee** [comment]
@thadG 

> Tried the image, which is not partitioned. Cannot get to the dietpi.txt
file to edit.

For ARMbian based images, we now use the default settings, everything is on one EXT4 partition, you'll need a EXT4 compatible system to edit the ```/boot/dietpi.txt``` file (eg: another linux system)

The only solution I can suggest is temporarily plug in HDMI + Keyboard, to setup the wifi in ```dietpi-config```, once completed you can resume headless.

**Comment by MichaIng** [comment]
@thadG 
Yeah, there are also some tools that provide ext4 support for Windows, but they are working not too reliable. A better solution is e.g. a USB stick with a live linux inside, that you can boot from your desktop: knoppix or a slimmer solution

**Comment by thadG** [comment]
Thanks,

Did not realize that.
I'll try to see if Ubuntu on Windows 10 can handle ext4. I'll also look
into placing knoppix into a USB.

**Comment by k-plan** [comment]
@thadG 
> Orange Pi One

What USB WiFi stick do you use? With which chipset?


@Fourdee 
> For ARMbian based images, we now use the default settings, **everything is on one EXT4 partition**, ...

Really?  😳   ... what a crappy mess ... :upside_down_face:    😟  👎 
Sorry. 

.


 (_k-plan - now discontinue support for such devices since v.6 ..._) 🙈

**Comment by Fourdee** [comment]
@k-plan 

> Really? 😳 ... what a crappy mess

I agree, however, i read somewhere ARMbian ```BOOTSIZE``` setting was depreciated, and, didn't want the hassle of their support, if i'am completely honest.

Defaults it is.

**Comment by Fourdee** [comment]
Closed in preference of pending 1MB fat https://github.com/Fourdee/DietPi/issues/1423#issue-291560616
